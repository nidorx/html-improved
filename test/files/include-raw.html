<!DOCTYPE html>
<html>
    <head>
        <title>test</title>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <include file="./includes/sample.js"/>
        <include file="./includes/sample.css"/>
    </head>
    <body>
        <h1>#{otherVar}</h1>

        <block name="body" />
    </body>
</html>
