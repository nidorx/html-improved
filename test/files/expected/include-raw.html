<!DOCTYPE html>
<html>
    <head>
        <title>test</title>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <script type="text/javascript">
            (function () {
                alert('hellow');
            })();
        </script>
        <style type="text/css">
            body, html {
                background: #ff0000;
            }
        </style>
    </head>
    <body>
        <h1>#{otherVar}</h1>
    </body>
</html>
